omnibus/resources/agent/msi/assets/sitedlg.wxi

```diff
@@ -6,6 +6,7 @@
     <ListItem Text="us5.datadoghq.com" Value="us5.datadoghq.com"/>
     <ListItem Text="ddog-gov.com" Value="ddog-gov.com"/>
     <ListItem Text="ap1.datadoghq.com" Value="ap1.datadoghq.com"/>
+    <ListItem Text="ap2.datadoghq.com" Value="ap2.datadoghq.com"/>
```

Comment:
this is the pre7.47 installer code, we don't need to modify it.
It wasn't removed due to a bug in the linters that failed the linters when too many files/lines were changed. Maybe that's fixed now...